org.civicrm.bootstrapcivihr/scss/components/outside-namespace/_h-scroll-shadows.scss

```diff
@@ -0,0 +1,42 @@
+.chr_h-scroll-shadows__master {
+  overflow: hidden; // this ensures the shadows are hidden behind
+  position: relative;
+}
+
+.chr_h-scroll-shadows__wrapper {
```

Comment:
i'm not sure these are the most descriptive names we can find, what about something like
```
.chr_h-scroll-shadows__wrapper
  .chr_h-scroll-shadows__shadows
    .chr_h-scroll-shadows__content 
```

Reply:
I was thinking about that.
.chr_h-scroll-shadows__shadows - is not "shadows", it is a wrapper that holds the __content inside so it makes it scrollable.